Source/Core/Core/IPC_HLE/WII_IPC_HLE.h

```diff
@@ -58,6 +58,7 @@ void SetDefaultContentFile(const std::string& _rFilename);
 void ES_DIVerify(const std::vector<u8>& tmd);
 
 void SDIO_EventNotify();
+void SDIO_EventNotify_CPUThread(u64 userdata, s64 cycles_late);
```

Comment:
Why are you declaring it in a .h if it's never used outside? Make it static / within an unnamed namespace.